src/Selenium2Library/keywords/_element.py

```diff
@@ -272,6 +272,30 @@ def click_element(self, locator):
         self._info("Clicking element '%s'." % locator)
         self._element_find(locator, True, True).click()
 
+    def key_click_element(self, keyname, locator):
```

Comment:
[Thinking out loud before researching as reminder to self] Should the order of the arguments be self, locator, keyname to be consistant with the other keywords?

Also I am thinking that we should add an option argument for specifying which button to click.